frontend/src/metabase/query_builder/components/view/sidebars/SummarizeSidebar/SummarizeContent/SummarizeBreakoutColumnList.tsx

```diff
@@ -33,7 +32,8 @@ export const SummarizeBreakoutColumnList = ({
     spacing="0"
     {...containerProps}
   >
-    <SectionTitle>{t`Group by`}</SectionTitle>
+    <Title order={5} fw={900}>{t`Group by`}</Title>
+    <Space my="sm" />
```

Comment:
and `SectionTitle` had a `margin-bottom: 1rem` - a `Space` with `my=sm` creates a `1rem` space